client/ayon_resolve/plugins/publish/extract_workfile.py

```diff
@@ -16,29 +18,22 @@ class ExtractWorkfile(publish.Extractor):
     hosts = ["resolve"]
 
     def process(self, instance):
-        # create representation data
-        if "representations" not in instance.data:
-            instance.data["representations"] = []
-
-        name = instance.data["name"]
         project = instance.context.data["activeProject"]
-        staging_dir = self.staging_dir(instance)
 
-        ext = ".drp"
-        drp_file_name = name + ext
-        drp_file_path = os.path.normpath(
-            os.path.join(staging_dir, drp_file_name))
+        host = registered_host()        
+        drp_file_path = host.get_current_workfile()
```

Comment:
We could also just get the already collected path from here https://github.com/ynput/ayon-resolve/pull/15/files#diff-d7c7c39f26e05a52a09cc2380f7dbe5c82a160e83ed1118b6fb7cca9925666c7R30

Reply:
Good call, thanks for that.